common/include/cstddef

```diff
@@ -0,0 +1,89 @@
+#ifndef _CSTDDEF_
+#define _CSTDDEF_
+#include <stddef.h>
+#include <type_traits>
+
+namespace std
+{
+using size_t = ::size_t;
+using ptrdiff_t = ::ptrdiff_t;
+using max_align_t = ::max_align_t;
+using nullptr_t = decltype(nullptr);
+
+enum class byte : unsigned char
```

Comment:
Braces on same line as enum/struct/class definitions; here and throughout the PR.